pkg/auth/model/model.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+option go_package = "github.com/treeverse/lakefs/auth/model";
+
+import "google/protobuf/timestamp.proto";
+
+package io.treeverse.lakefs.auth.model;
+
+message UserData {
+    int64 ID = 1;
+    google.protobuf.Timestamp CreatedAt = 2;
+    string Username = 3;
+    // FriendlyName, if set, is a shorter name for the user than
+    // Username.  Unlike Username it does not identify the user (it
+    // might not be unique); use it in the user's GUI rather than in
+    // backend code.
+    string FriendlyName = 4;
+    string Email = 5;
+    bytes EncryptedPassword = 6;
+}
+
+message GroupData {
+    int64 ID = 1;
```

Comment:
For each go struct that holds an 'int' value - since it is an arch specific type we need to decid whether it is int32 or int64 and define it explicitly in the struct - otherwise we might encountered unexpected behavior

Reply:
The problem is that int is not a specific type, it can mean several types and depends on the system. Think about a situation where you save an int64 ID into the database on a 64 arch system, you export it and then try to import it in a 32bit arch -> this could potentially overflow. We should take this opportunity to set specific types in the structs